scripts/ci_test_brownie.sh

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env bash
 
-pip install eth-brownie
+# Install eth-brownie if it's not already present
+if [[ -z "$(command -v brownie)" ]]
+then pip install eth-brownie
+fi
+
 brownie bake token
-cd token || exit 255
 
-crytic-compile . --compile-force-framework Brownie
+cd token || exit 255
 
-if [ $? -ne 0 ]
-then
-    echo "Brownie test failed"
-    exit 255
+if ! crytic-compile . --compile-force-framework Brownie
+then echo "Brownie test failed" && exit 255
+else echo "Brownie test passed" && exit 0
```

Comment:
Unrelated to core issue resolutions: I used `shellcheck` to lint all CI test scripts

Reply:
nvmd, CI broke so hopefully reverting these changes will help debug automated tests